examples/kubernetes/connectivity-check/tools.cue

```diff
@@ -0,0 +1,34 @@
+package connectivity_check
+
+// Definitions in this file are not intended for continuous integration
+// deployment, as their readiness/liveness probes are overridden to always
+// pass. Instead, they can be deployed and their logs followed for manual
+// testing via commands like the following:
+//
+// $ kubectl logs -l name=query-dns-policy --timestamps -f
+
+// query-dns-policy is written for debugging by watching the logs; the
+// liveness/readiness probes are bypassed so it will always report success!
+deployment: "query-dns-policy": {
+	_image: "alpine"
+	_command: ["/bin/sh", "-c", "apk update && apk add bind-tools && while true; do dig +noall +question +answer +timeout=1 +tries=1 www.google.com && sleep 1 ; done"]
```

Comment:
If anyone has suggestions for a simple container image that includes `dig`, we could substitute that here & drop the install step.

Reply:
just curious if nslookup can be used here as well, then we can avoid apk commands, and remove related alpine egress policy.

Alternatively, I observed that we are using one image called alpine-curl in connectivity test, any plan to naintain one alpine image with (bind-tools + netutils + curl) for debugging purpose like what you have here as well ?

Reply:
`nslookup` doesn't have the features we need for this test.

Looks like we can use `cilium/demo-client` docker image instead, it already has all the tools we need.